deploy/terraform/terraform-units/modules/sap_system/common_infrastructure/infrastructure.tf

```
@@ -35,6 +35,46 @@ data "azurerm_virtual_network" "vnet-sap" {
   resource_group_name = split("/", local.vnet_sap_arm_id)[4]
 }
 
+// Creates admin subnet of SAP VNET
+resource "azurerm_subnet" "admin" {
+  count                = local.sub_admin_exists ? 0 : local.create_admin_subnet ? 1 : 0
+  name                 = local.sub_admin_name
+  resource_group_name  = local.rg_exists ? data.azurerm_resource_group.resource-group[0].name : azurerm_resource_group.resource-group[0].name
+  virtual_network_name = local.vnet_sap_exists ? data.azurerm_virtual_network.vnet-sap[0].name : azurerm_virtual_network.vnet-sap[0].name
+  address_prefixes     = [local.sub_admin_prefix]
+}
+
+# Imports data of existing SAP admin subnet
+data "azurerm_subnet" "admin" {
+  count                = local.sub_admin_exists ? 1 : 0
```

only import existing admin subnet when needed:
```suggestion
  count                = local.sub_admin_exists  && local.create_admin_subnet ? 1 : 0
```